src/components/Toolbar.js

```diff
@@ -10,13 +11,64 @@ const shortcutIds = [
   'search',
 ]
 
-export const Toolbar = connect(({ settings: { dark } }) => ({ dark }))(({ dark }) =>
-  <div className="toolbar">
-    {shortcutIds.map(id => {
-      const { name, svg: Icon, exec } = shortcutById(id)
-      return <div key={id} className="toolbar-icon" title={name} onClick={exec}>
-        <Icon fill={dark ? 'white' : 'black'} />
-      </div>
-    })}
-  </div>
-)
+const ToolbarMessageOverlay = ({ name, description }) => {
+    return (
```

Comment:
Parentheses are redundant. I also prefer implicit returns for single-line functions.

Reply:
I initially had a ToolbarMessageOverlay component but got rid of it, I must have forgotten to remove the parentheses.